### PR TITLE
Roadmap card fix [fixes #2195]

### LINKED ISSUE #2195: Roadmap cards overflow
**Describe the bug**

Card width don't handle text overflow gracefully, leaving to wacky layouts on long words:

Image 2020-12-31 at 12 20 50 PM

**To Reproduce**

1. Go to https://ethereum.org/en/about/
2. View "implemented features"

**Expected behavior**

Let's discuss & decide on a way to handle this, e.g. should we truncate the description w/ ellipses after a certain length? Or set `overflow` to `hidden` or `scroll`?

While we make this change we should also update these card styles to fit the cards on the [Languages page](https://ethereum.org/en/languages/):

Image 2020-12-31 at 12 26 33 PM
Perhaps this card style should be a component in SharedStyledComponents.js

**Additional context**

Issue that led to Language page card updates: #2076

### PR DESCRIPTION
**Pull request by ryancreatescopy**
## Description

updates roadmap cards and language cards
- consistent styling
- ensures consistent height of cards in same row

## Related Issue

fixes #2195

### CODE REVIEW

File: src/components/Roadmap.js [hunk 1]
```diff
@@ -14,27 +14,28 @@ const Section = styled.div`
...
   list-style: none;
-  border-radius: 0.5rem;
-  width: 100%;
-  border: 1px dotted ${(props) => props.theme.colors.lightBorder};
-  box-shadow: 0 1px 4px ${(props) => props.theme.colors.boxShadow};
-  transition: all 0.5s cubic-bezier(0.25, 0.8, 0.25, 1);
+  border-radius: 2px;
+  border: 1px solid ${(props) => props.theme.colors.lightBorder};
+  /*   box-shadow: ${(props) => props.theme.colors.tableBoxShadow};
+ */ /* transition: all 0.5s cubic-bezier(0.25, 0.8, 0.25, 1); */
```
Comment (samajammin) [inline src/components/Roadmap.js:29]: We should uncomment or delete these.

I thought the transition was pretty nifty... any particular reason you removed?

File: src/pages/languages.js [hunk 2]
```diff
@@ -16,34 +16,36 @@ const StyledPage = styled(Page)`
...
   padding: 1rem;
-  flex: 0 1 260px;
+  width: 240px;
   list-style: none;
   border-radius: 2px;
-  width: 100%;
   border: 1px solid ${(props) => props.theme.colors.lightBorder};
-  box-shadow: ${(props) => props.theme.colors.tableBoxShadow};
-  transition: all 0.5s cubic-bezier(0.25, 0.8, 0.25, 1);
+  /* box-shadow: ${(props) => props.theme.colors.tableBoxShadow};
```
Comment (samajammin) [inline src/pages/languages.js:37]: Same here.

**Review by samajammin — CHANGES_REQUESTED** [review]
Looks good! Minor comments.

**Comment by ryancreatescopy** [comment]
Sure, didn't want to delete until we agreed it was best approach. In terms of the transition, the blur felt quite extra to me haha. Either way we should agree on a "transition" style and make them consistent as possible across all cards on the site. The transition for these cards, for example, are different to the `ActionCard` components. I'll create a separate issue for that.

**Comment by ryancreatescopy** [comment]
Issue created for hover states: #2253

**Pull request merged**